File: src/test/java/cernx/colt/matrix/tfloat/impl/DenseLargeFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class DenseLargeFloatMatrix2DViewTest extends DenseLargeFloatMatrix2DTest {

    public DenseLargeFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseLargeFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseLargeFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseLargeFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}

File: src/test/scala/cern/colt/matrix/tfloat/impl/DenseLargeFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.DenseLargeFloatMatrix2D

class DenseLargeFloatMatrix2DViewTest(arg0: String) extends DenseLargeFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new DenseLargeFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new DenseLargeFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new DenseLargeFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
